### PR TITLE
Add qrc TechFile support and add to cadence library setup

### PR DESCRIPTION
**Pull request by colinschmidt**
Just to start the mmmc stuff going, still got a lot of other options to add

### CODE REVIEW

File: src/hammer-vlsi/hammer_vlsi.py [hunk 1]
```diff
@@ -1549,14 +1574,21 @@ def append_mmmc(cmd: str) -> None:
             list=library_set_name
         ))
         # extra junk: -opcond ...
+        rc_corner_name = "rc_cond"
+        append_mmmc("create_rc_corner -name {name} -temperature {temp} -qrc_tech {qrc}".format(
+            name=rc_corner_name,
+            temp=120, # TODO: this should come from tech config
+            qrc=self.get_qrc_tech()
```
Comment (edwardcwang) [inline src/hammer-vlsi/hammer_vlsi.py]: If there are no qrc techs, shouldn't we avoid emitting an empty "-qrc_tech" parameter?

**Comment by edwardcwang** [comment]
@colinschmidt Is there more in this series?

File: src/hammer-vlsi/hammer_vlsi.py [hunk 2]
```diff
@@ -1549,14 +1574,21 @@ def append_mmmc(cmd: str) -> None:
             list=library_set_name
         ))
         # extra junk: -opcond ...
+        rc_corner_name = "rc_cond"
+        append_mmmc("create_rc_corner -name {name} -temperature {temp} -qrc_tech {qrc}".format(
+            name=rc_corner_name,
+            temp=120, # TODO: this should come from tech config
```
Comment (edwardcwang) [inline src/hammer-vlsi/hammer_vlsi.py]: What are the units of this `temp`?

File: src/hammer-vlsi/hammer_vlsi.py [hunk 3]
```diff
@@ -849,6 +849,21 @@ def extraction_func(lib: hammer_tech.Library) -> List[str]:
         return LibraryFilter.new("timing_lib", "CCS/NLDM timing lib (liberty ASCII .lib)",
                                  extraction_func=extraction_func, is_file=True)
 
+    @property
+    def qrc_tech_filter(self) -> LibraryFilter:
+        """
+        Selecting qrc RC Corner tech (qrcTecch) files.
```
Comment (edwardcwang) [inline src/hammer-vlsi/hammer_vlsi.py]: Typo?

**Comment by colinschmidt** [comment]
No this is just the bare support, so nothing more yet.

File: src/hammer-vlsi/hammer_vlsi.py (hunk 2 above)
Comment (colinschmidt) [reply src/hammer-vlsi/hammer_vlsi.py]: Celsius. Do you want me to note that?

Comment (edwardcwang) [reply src/hammer-vlsi/hammer_vlsi.py]: Yeah that make sense and would probably be nice to document.

**Comment by colinschmidt** [comment]
Fixed comments

**Comment by colinschmidt** [comment]
Let me know if there is anything else. Thanks

**Pull request merged**